src/components/contacts/Contacts.js

```diff
@@ -1,8 +1,8 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
```

Comment:
Hell yeah , useRef!!!

Reply:
Learning has occurred! 🧠

Reply:
How is useRef being used in this instance?

Reply:
I assigned `useRef` to a variable that works as a two-street for the `input` tag's `value`. So that variable is a `reference` to whatever the `value` is of the search bar `input`.